pkg/client/registry.go

```diff
@@ -152,6 +153,10 @@ func RegistryConnectNetworks(ctx context.Context, runtime runtimes.Runtime, regi
 	// get cluster details and connect
 	failed := 0
 	for _, net := range networks {
+		if util.ContainsString(registryNode.Networks, net) {
+			log.Infof("Container '%s' is already connected to '%s'", registryNode.Name,net)
+			continue
+		}
 		if err := runtime.ConnectNodeToNetwork(ctx, registryNode, net); err != nil {
```

Comment:
Somehow I have the feeling, that this function here should just not throw an error, if the node/container is already connected to this network, right?

Reply:
Hi, thank you for review. Changes amended.